javascripts/htmlBuilder.js

```diff
@@ -0,0 +1,51 @@
+
+//**********//HTML BUILDER//**********//
+var petFood = function(object) {
+	
+	//DECLARES VARIABLE FOR OUTPUT DOM ELEMENT//
+	var mainContent = document.getElementById("mainContent");
+
+	//BUILDS HTML CONTENT FROM ARRAY PARAMETER FOR A GIVEN ANIMAL//
+	object.htmlBuilder = function(array, foodAnimal) {
+
+		//ADDS FOOD ANIMAL HEADER//
+		content = `<ul class="foodAnimal">`
+		content += `<li><h1>${foodAnimal.toUpperCase()}</h1></li>`;
+
+		//ADDS BRAND NAME//
+		for (var i = 0; i < array.length; i++) {
+			content += '<ul class="foodBrand">';
+			content += `<li><h2>${array[i].name}</h2></li>`;
+
```

Comment:
Not a necessary change, but a helpful tactic when dealing with nested loops, once you're in a layer of the loop, you can save off the element you're at to a variable, and then use that further down the loop chain instead of having to do stuff like ```array[i].arrayTwo[j].arrayThree[k]...```

For instance, if you did something like ```var brand = array[i]``` you could then use that further down...